os/dbuild.sh

```diff
@@ -153,6 +154,10 @@ function SELECT_OPTION()
 		x|exit)
 			exit 1
 			;;
+		l|tool)
+			read TL_ARGS
+			TOOLCHAIN tool $TL_ARGS
+			;;
```

Comment:
This is a menu so that how about providing new selectable options for addr2line, nm and manual input?
If we provide addr2line and nm, then we can automatically input an elf file path, and get an address only.

Reply:
Done. Thanks.